packaging/hudi-utilities-slim-bundle/pom.xml

```diff
@@ -109,6 +109,8 @@
 
                   <include>com.github.davidmoten:guava-mini</include>
                   <include>com.github.davidmoten:hilbert-curve</include>
+                  <!-- SPARK-43489 Spark 3.5+ has marked protobuf as provided -->
+                  <include>com.google.protobuf:protobuf-java</include>
```

Comment:
This is needed otherwise deltastreamer would fail due to
```
Exception in thread "main" java.lang.NoClassDefFoundError: com/google/protobuf/Message
    at java.lang.Class.getDeclaredMethods0(Native Method)
    at java.lang.Class.privateGetDeclaredMethods(Class.java:2729)

Reply:
This should not be included, as the `hudi-utilities-slim-bundle` should be used with `hudi-spark-bundle`, which contains `com.google.protobuf:protobuf-java`?

Reply:
I don't see `hudi-spark-bundle` including `com.google.protobuf:protobuf-java` now. But yeah I agree if protobuf is in spark bundle then we shouldn't include it in the slim bundle